src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/ProfilerEditingTarget.java

```diff
@@ -70,6 +106,17 @@ public ProfilerEditingTarget(ProfilerPresenter presenter,
       server_ = server;
       globalDisplay_ = globalDisplay;
       pSourceWindowManager_ = pSourceWindowManager;
+      fileDialogs_ = fileDialogs;
+      fileContext_ = fileContext;
+      workbenchContext_ = workbenchContext;
+      eventBus_ = eventBus;
+      sourceServer_ = sourceServer;
+      
+      if (!initializedEvents_)
```

Comment:
It's possible since this initializes `static` event shared across all profilers, this is used to handle the `window.postMessage` entry point.